shell/platform/windows/flutter_windows_view.cc

```diff
@@ -127,6 +129,14 @@ void FlutterWindowsView::SendWindowMetrics(size_t width,
   auto result = FlutterEngineSendWindowMetricsEvent(engine_->engine(), &event);
 }
 
+void FlutterWindowsView::SendInitialBounds() {
+  // Initial bounds must be sent after engine has initialized
+  PhysicalWindowBounds bounds = binding_handler_->GetPhysicalWindowBounds();
+
+  SendWindowMetrics(bounds.width, bounds.height,
+                    binding_handler_->GetDpiScale());
```

Comment:
Isn't this new method always what we want sending window metrics to do? It should never be the case that we tell Flutter something that doesn't match the actual backing store, I would think.

If that's true, then we can just have a single `SendUpdatedWindowMetrics()` instead of these two variants.

Reply:
Doing so would introduce an unnecessary call to get physical bounds in the resize case where the bounds have already been derived from the physical window resizing plumbing (in win32 case, the lparam of WM_SIZE)